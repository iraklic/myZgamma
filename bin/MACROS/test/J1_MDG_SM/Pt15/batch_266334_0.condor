000 (266334.000.000) 09/20 00:30:26 Job submitted from host: <131.225.191.209:56594>
...
001 (266334.000.000) 09/20 00:32:03 Job executing on host: <131.225.191.147:59038>
...
006 (266334.000.000) 09/20 00:32:11 Image size of job updated: 82096
...
005 (266334.000.000) 09/20 03:35:24 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 03:02:53, Sys 0 00:00:02  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 03:02:53, Sys 0 00:00:02  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	108009  -  Run Bytes Sent By Job
	23746296  -  Run Bytes Received By Job
	108009  -  Total Bytes Sent By Job
	23746296  -  Total Bytes Received By Job
...
