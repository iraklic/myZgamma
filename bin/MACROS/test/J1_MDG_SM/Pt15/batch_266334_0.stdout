
CMSSW on Condor

started at Tue Sep 20 00:32:03 CDT 2011

parameter set:
CONDOR_CLUSTER: 266334
CONDOR_PROCESS: 0
CMSSW_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_4_2_0
RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam
PARAMETER_SET: output_Mu_B_S.txt
NUM_EVENTS_PER_JOB: 

Writing final parameter-set: batch_266334_0.py to RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam

run: myMinexam > batch_266334_0.log 2>&1\n

finished at Tue Sep 20 03:35:24 CDT 2011
